spec/models/education_experience_spec.rb

```diff
@@ -38,4 +38,18 @@
       it { expect(education_experience).to_not be_valid }
     end
   end
+
+  describe '.for_user' do
+    let!(:user) { create(:user) }
+    let!(:education_experience_from_user) { create(:education_experience, user:) }
+    let!(:education_experience_from_another_user) { create(:education_experience) }
+
+    it 'returns education experiences associated with the specified user' do
+      expect(EducationExperience.for_user(user.id)).to include(education_experience_from_user)
```

Comment:
You should match the correct response, only checking if it includes the experience, does not verify the correct return, you should verify the full return